.github/workflows/tests-rs-package.yml

```diff
@@ -62,10 +62,11 @@ jobs:
         uses: ./.github/actions/rust
         with:
           components: rustfmt
+          toolchain: nightly
           cache: false
 
       - name: Check formatting
-        run: exit `cargo fmt --check --package=${{ inputs.package }} | wc -l`
+        run: exit `cargo +nightly fmt --check --package=${{ inputs.package }} | wc -l`
```

Comment:
How is it going to work since we are still on stable channel and my IDE formats changed files automatically?